app/models/kolide/issue.rb

```diff
@@ -7,6 +7,14 @@ class Issue < ::ActiveRecord::Base
     belongs_to :device
     belongs_to :check
 
+    def identifier
```

Comment:
Maybe this should be named `id_md` or `identifier_markdown` or something like that.